.github/renovate.json

```
@@ -57,6 +57,10 @@
       "groupName": "engines",
       "matchDepTypes": ["engines"],
       "labels": ["Engines"]
+    },
+    {
+      "matchPackageNames": ["node"],
+      "allowedVersions": "<21.0.0"
```

OK for me, but to be consistent with the other groups, I suggest

```suggestion
    {
      "groupName": "node LTS",
      "matchPackageNames": ["node"],
      "allowedVersions": "<21.0.0",
      "labels": ["Engines"]
```